javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -139,7 +135,8 @@ public Map<Request, List<ProxyMethod>> map(Operation operation) {
             builder.returnType(GenericType.Mono(clientResponseClassType));
         } else {
             IType singleValueType;
-            if (responseBodyType.equals(GenericType.FluxByteBuffer)) {
+            if (responseBodyType.equals(GenericType.FluxByteBuffer)
```

Comment:
wondering how we get `responseBodyType = GenericType.FluxByteBuffer` in old code...

Reply:
no idea either... I'll set break point and run generate.bat to see if it ever stops...

Reply:
OK to leave it be. Just curious, since `responseBodyType` should be sync, but Flux is async.

Reply:
No luck in generate.bat, I'll leave it be.